Library/Homebrew/cmd/audit.rb

```diff
@@ -1250,6 +1253,19 @@ def audit_urls
       end
     end
 
+    # Debian has an abundance of secure mirrors. Let's not pluck the insecure
+    # one out of the grab bag.
+    urls.each do |u|
+      next unless u =~ %r{^http://http\.debian\.net/debian/(.*)}i
+      problem <<-EOS.undent
+        Please use a secure mirror for Debian URLs.
+        We recommend either:
```

Comment:
Given it's a `mirror` I think it's fine to standardise on one. Regardless, those mirrors aren't hurting for bandwidth.